tests/features/CreatePostsTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+class CreatePostsTest extends FeaturesTestCase
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)

<!-- Comment by [SideCI](https://sideci.com) -->